main.html

```diff
@@ -10,9 +10,10 @@
   <link rel="stylesheet" type="text/css" href="style/drawer.css">
 
   <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1">
+  <script type="text/javascript" src="midlet.js" defer></script>
   <!-- Uncomment the line below to enable the profiler. -->
   <!--<script type="text/javascript" src="build/shumway.js" defer></script>-->
-  <script type="text/javascript" src="midlet.js" defer></script>
+  <!--<script type="text/javascript" src="libs/terminal.js" defer></script>-->
```

Comment:
This new terminal looks handy, but since it's disabled, and since it's orthogonal to the purpose of this branch, can it be split into its own pull request?